wowchemy-cms/assets/js/shortcodes-cms.js

```diff
@@ -0,0 +1,186 @@
+CMS.registerEditorComponent({
```

Comment:
Adds support for a few shortcodes within the CMS.
See: https://www.netlifycms.org/docs/custom-widgets/
Based on https://github.com/sharadcodes/hugo-shortcodes-netlify-cms
but adds support for the callout and figure shortcodes.